macros/helpers/nozzle_cleaning.cfg

```diff
@@ -51,15 +51,7 @@ gcode:
 
     G90
 
-    {% if purge_and_brush_enabled %}
-        # Move to purge zone only if it's available, else just purge whe the toolhead is
-        {% set Px, Py, Pz = printer["gcode_macro _USER_VARIABLES"].purge_bucket_xyz|map('float') %}
-        {% if Z_DROP == 1 %}
-            G1 X{Px} Y{Py} Z{Pz} F{St}
-        {% else %}
-            G1 X{Px} Y{Py} F{St}
-        {% endif %}
-    {% endif %}
+    MOVE_TO_PURGE_BUCKET
```

Comment:
Be careful to also pass the Z_DROP parameter as I use it when doing an M600 to purge after changing the filament while keeping the Z at the current Z (without falling down at a small Z risking to hit something: this is the goal of the optional Z_DROP)

Reply:
parameter added